toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/panels/SimplePanel.java

```diff
@@ -0,0 +1,36 @@
+package io.toast.tk.agent.ui.panels;
+
+import io.toast.tk.agent.ui.utils.PanelHelper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.swing.*;
+import javax.swing.border.TitledBorder;
+import java.awt.*;
+import java.io.IOException;
+import java.util.Properties;
+
+
+public class SimplePanel extends AbstractPanel {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This class has 6 parents which is greater than 5 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)